assets/js/blocks/cart-checkout/cart/full-cart/style.scss

```diff
@@ -1,7 +1,7 @@
 .wc-block-cart {
 	color: $core-grey-dark-600;
 
-	h2 {
+	.wc-block-cart__title {
```

Comment:
Since we're setting the specificity of a "title" element, I wonder if we should make this a more generic selector that could be used across any of our blocks that might have this kind of title element (which we want to retain some measurement of control over). In other words, I wonder if this is _too_ specific of a name. Could we just use something like `.wc-block__title` instead and make that a more "global" selector used anyplace we want this specific styled title?

Reply:
Yeah, good point. In e0d0e2aace64d1529a55651f69c228152ea128fc I created a `<Title>` component so styles are contained there instead of having a generic class without an associated component. Let me know how it looks.